web/src/views/FileBrowserView/FileBrowser.vue

```diff
@@ -396,12 +404,11 @@ function locationSlice(index: number) {
   return `${splitLocation.value.slice(0, index + 1).join('/')}/`;
 }
 
-function selectPath(item: AssetPath) {
-  const { asset, path } = item;
-
-  // Return early if path is a file
-  if (asset) { return; }
-  location.value = path;
+function itemIsViewable(item: AssetPath) {
+  const nonViewableExtensions = ['nwb', 'zip', 'gz', 'nii', 'tif', 'tiff', 'avi'];
```

Comment:
I was surprised to discover that `.tiff` is not supported by most of the browsers.  It is supported by [Safari though](https://en.wikipedia.org/wiki/Comparison_of_web_browsers#Image_format_support).  But I also wonder if/how it would interact with some browser extensions which might be there to support viewing those types like https://www.blackice.com/broadcast/10132020TIFFV/index.html ?